packages/vue/tests/unit/AdvancedVideo.spec.ts

```diff
@@ -110,6 +115,28 @@ describe("AdvancedVideo", () => {
     expect(component.html()).toContain('poster="www.example.com"');
   });
 
+  it('should contain poster when "auto" is passed as cldPoster', async function () {
+    const component = mount(AdvancedVideo, {
+      props: { cldVid: cloudinaryVideo, cldPoster: "auto" },
+    });
+    await waitTicks(1);
+
+    expect(component.html()).toContain(
+      'poster="https://res.cloudinary.com/demo/video/upload/q_auto/f_jpg/so_auto/sample"'
+    );
+  });
+
+  it("should contain poster when cloudinary image is passed as cldPoster", async function () {
+    const component = mount(AdvancedVideo, {
+      props: { cldVid: cloudinaryVideo, cldPoster: cloudinaryImage },
+    });
+    await waitTicks(1);
+
+    expect(component.html()).toContain(
+      'poster="https://res.cloudinary.com/demo/image/upload/sample"'
```

Comment:
maybe instead of hardcoded string call `cloudinaryImage.toUrl()?`

Reply:
This sounds good! 

I see we're actually asserting the string value of `https://res.cloudinary.com/demo/image/upload/sample` in multiple tests all over the place. That is something we might fix as a dev task globally FYI @lbendavid-cloudinary